HHS/Head-Start-TTADP
# [TTAHUB-430] add approved and created date to activity report tables

## Description of change
- Added approvedAt as a database column. This is set to the most recent timestamp when the report status changed to approved. Otherwise it is null. Unlocking a report doesn't erase this column, but re-approving it afterwards does overwrite it.
- Added created date to the my alerts widget on the activity-reports page
- Added created date and approved date to the activity reports widget
- Added both fields to the CSV (the same function generates exports for both tables - if there isn't an approval date, it will be blank)

## How to test
View the tables on /activity-reports and export CSVs. Check that the fields described above are displayed, and that everything else appears correct.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-430

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
